spring-data-sample/README.md

```diff
@@ -0,0 +1,262 @@
+
+You need to run the following `docker-compose` command:
+```shell
+$ docker-compose up -d
+```
+
+This command starts Cassandra and loads the schema.
+Please note that you need to wait around more than one minute for the containers to be fully started.
+
+### Initial data
+
+You first need to load initial data with the following command:
+```shell
+$ ./gradlew run --args="LoadInitialData"
+```
+
+And the following data will be loaded:
+
+- For the `sample.customers` table:
+
+| customer_id | name          | credit_limit | credit_total |
+|-------------|---------------|--------------|--------------|
+| 1           | Yamada Taro   | 10000        | 0            |
+| 2           | Yamada Hanako | 10000        | 0            |
+| 3           | Suzuki Ichiro | 10000        | 0            |
+
+- For the `sample.items` table:
+
+| item_id | name   | price |
+|---------|--------|-------|
+| 1       | Apple  | 1000  |
+| 2       | Orange | 2000  |
+| 3       | Grape  | 2500  |
+| 4       | Mango  | 5000  |
+| 5       | Melon  | 3000  |
+
+## Run the sample application
+
+Let's start with getting the customer information whose ID is `1`:
+```shell
+$ ./gradlew run --args="GetCustomerInfo 1"
+...
+{"customer_id":1,"name":"Yamada Taro","credit_limit":10000,"credit_total":0}
+...
+```
+
+Then, place an order for three apples and two oranges with customer ID `1`. Note that the format of order is `<Item ID>:<Count>,<Item ID>:<Count>,...`:
+```shell
+$ ./gradlew run --args="PlaceOrder 1 1:3,2:2"
+...
+{"order_id":"2358ab35-5819-4f8f-acb1-12e73d97d34e","customer_id":1,"timestamp":1677478005400}
+...
+```
+
+The command shows the order ID of the order.
+
+Let's check the details of the order with the order ID:
+```shell
+$ ./gradlew run --args="GetOrder dea4964a-ff50-4ecf-9201-027981a1566e"
+...
+{"order_id":"2358ab35-5819-4f8f-acb1-12e73d97d34e","timestamp":1677478005400,"customer_id":1,"customer_name":"Yamada Taro","statements":[{"item_id":5,"item_name":"Melon","price":3000,"count":1,"total":3000}],"total":3000}
+...
+```
+
+So, let's place an order again and get the order histories by customer ID `1`:
+```shell
+$ ./gradlew run --args="PlaceOrder 1 5:1"
+...
+{"order_id":"46062b16-b71b-46f9-a9ff-dc6b0991259b","customer_id":1,"timestamp":1677478201428}
+...
+$ ./gradlew run --args="GetOrders 1"
+...
+[{"order_id":"2358ab35-5819-4f8f-acb1-12e73d97d34e","timestamp":1677478005400,"customer_id":1,"customer_name":"Yamada Taro","statements":[{"item_id":5,"item_name":"Melon","price":3000,"count":1,"total":3000}],"total":3000},{"order_id":"46062b16-b71b-46f9-a9ff-dc6b0991259b","timestamp":1677478201428,"customer_id":1,"customer_name":"Yamada Taro","statements":[{"item_id":5,"item_name":"Melon","price":3000,"count":1,"total":3000}],"total":3000}]
+...
+```
+
+These histories are ordered by timestamp in a descending manner.
+
+The current `credit_total` is `10000`, so it has reached the `credit_limit`.
+So, the customer can't place an order anymore due to the limit.
```

Comment:
```suggestion
The customer's current `credit_total` is `10000`. Since the customer has now reached their `credit_limit`, which was shown when retrieving their information, they cannot place anymore orders.
```